diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LambdaLengthRule.kt

```diff
@@ -0,0 +1,71 @@
+package org.cqfn.diktat.ruleset.rules
+
+import org.cqfn.diktat.common.config.rules.RuleConfiguration
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.common.config.rules.getRuleConfig
+import org.cqfn.diktat.ruleset.constants.EmitType
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.utils.*
+
+import com.pinterest.ktlint.core.Rule
+import com.pinterest.ktlint.core.ast.ElementType
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+
+/**
+ * Rule 5.2.4 check lambda length without parameters
+ */
+class LambdaLengthRule(private val configRules: List<RulesConfig>) : Rule("lambda-length") {
+    private val configuration by lazy {
+        LambdaLengthConfiguration(
+            this.configRules.getRuleConfig(Warnings.TOO_MANY_LINES_IN_LAMBDA)?.configuration ?: emptyMap()
+        )
+    }
+    private var isFixMode: Boolean = false
+    private lateinit var emitWarn: EmitType
+
+    override fun visit(
+        node: ASTNode,
+        autoCorrect: Boolean,
+        emit: EmitType
+    ) {
+        emitWarn = emit
+        isFixMode = autoCorrect
+
+        if (node.elementType == ElementType.LAMBDA_EXPRESSION) {
+            checkLambda(node, configuration)
+        }
+    }
+
+    private fun checkLambda(node: ASTNode, configuration: LambdaLengthConfiguration) {
+        val copyNode = node.clone() as ASTNode
+        val sizeLambda = countCodeLines(copyNode)
+        if (sizeLambda > configuration.maxLambdaLength) {
+            copyNode.findAllNodesWithCondition({it.elementType == ElementType.LAMBDA_EXPRESSION}).forEachIndexed {index, node ->
```

Comment:
```suggestion
            copyNode.findAllNodesWithCondition({ it.elementType == ElementType.LAMBDA_EXPRESSION }).forEachIndexed { index, node ->
```
We usually use spaces in lambdas after `{` and before `}`, that's a bug that diktat doesn't check them, I already opened an issue for this.